crates/fondue/out/index.js

```
@@ -1,69 +1,512 @@
-/* tslint:disable */
+// prettier-ignore
 /* eslint-disable */
-/* prettier-ignore */
-
+// @ts-nocheck
 /* auto-generated by NAPI-RS */
 
-const { existsSync, readFileSync } = require('fs')
-const { join } = require('path');
+const { createRequire } = require('node:module')
+require = createRequire(__filename)
```

Biome (2.1.2)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)